lib/Flux.js

```diff
@@ -126,18 +215,36 @@ class Flux {
     return first;
   }
 
+  /**
+   * Fetch the content of a store given his path.
+   * @param {*} binding TODO: To implement after signature change.
```

Comment:
I don't think putting TODOs is relevant. When we'll change the function signature we'll also change the function description accordingly.